src/nodes.coffee

```diff
@@ -765,6 +765,10 @@ exports.CSXTag = class CSXTag extends IdentifierLiteral
 exports.PropertyName = class PropertyName extends Literal
   isAssignable: YES
 
+exports.ComputedPropertyName = class ComputedPropertyName extends PropertyName
+  compileNode: (o) ->
+    [@makeCode "[" + @value.compile(o) + "]"]
```

Comment:
These `+`s are casting whatever is output by `@value.compile(o)` as a string. Does `compile` always return a string?

I feel like this would be safer:
```coffee
  compileNode: (o) ->
    [@makeCode('['), @value.compile(o), @makeCode(']')]
```
Unless `@value.compile` should be `@value.compileToFragments` . . . ?

Reply:
Thanks. It's changed to   
`[@makeCode('['), @value.compileToFragments(o, LEVEL_LIST)..., @makeCode(']')]`